shttps/Server.cpp

```diff
@@ -721,6 +774,11 @@ namespace shttps {
     }
     //=========================================================================
 
+    // process request is one thread that is waiting for incoming requests
+    // and processes them. It is started by the main thread and will run
+    // until the server is stopped.
+    // This is in addition to the thread pool that is used to process the
+    // lua scripts.
```

Comment:
does this mean that all non-lua logic is single threaded?

Reply:
no, everything is multithreaded and this comment is a bit misleading. There is only one thread pool used by all routes. I will correct it.

Reply:
@BalduinLandolt can you maybe take again a look at the changes?

Reply:
> does this mean that all non-lua logic is single threaded?

also please note the new naming of the methods. Before we had `process_request` and `processRequest`. I hope the new naming is a bit more clearer what is happening (at least what I think that is happening).